cmd/clusterctl/client/config.go

```diff
@@ -51,8 +52,12 @@ func (c *clusterctlClient) GetProviderComponents(provider string, providerType c
 
 // GetClusterTemplateOptions carries the options supported by GetClusterTemplate.
 type GetClusterTemplateOptions struct {
-	// Kubeconfig file to use for accessing the management cluster. If empty, default discovery rules apply.
-	Kubeconfig string
+	// KubeconfigPath defines the path to the kubeconfig file to use for accessing the management cluster. If empty, default discovery rules apply.
+	KubeconfigPath string
```

Comment:
```suggestion
	Kubeconfig cluster.Kubeconfig
```
Might be we should alias the structure as well so we can avoid exposing the cluster package